samples/databases/wide-world-importers/wwi-ssdt/wwi-ssdt/DataLoadSimulation/Tables/ColdRoomTemperatures_temp.sql

```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE DataLoadSimulation.[ColdRoomTemperatures_temp] (
+    [ColdRoomTemperatureID] BIGINT                                      NOT NULL,
+    [ColdRoomSensorNumber]  INT                                         NOT NULL,
+    [RecordedWhen]          DATETIME2 (7)                               NOT NULL,
+    [Temperature]           DECIMAL (10, 2)                             NOT NULL,
+    [ValidFrom]             DATETIME2 (7) NOT NULL,
+    [ValidTo]               DATETIME2 (7) NOT NULL,
+    INDEX [IX_DataSimulation_ColdRoomTemperatures_ColdRoomSensorNumber] 
+		NONCLUSTERED HASH ([ColdRoomSensorNumber]) WITH (BUCKET_COUNT=100000)
```

Comment:
Nice to have: 
Since bucket_count is a sore spot, we should probably add a nice comment here indicating why we chose this bucket count (e.g., we estimate these many rows etc...hence we choose this bucket count)